xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -172,6 +173,7 @@ public boolean shouldAccept(Runnable command) {
 
   @Before
   public void setUp() throws IOException {
+    ClientXdsClient.enableFaultInjection = true;
```

Comment:
Wait, this is lazy. You got to do it correctly.

Reply:
This is following the same style as `enableTimeout ` https://github.com/grpc/grpc-java/blob/v1.35.0/xds/src/test/java/io/grpc/xds/XdsNameResolverTest.java#L117

Reply:
I mean you need to turn it back, and cover the default/unset behavior.

Reply:
Exactly I mean `enableTimeout` was not turned back.

Reply:
Oh, really? Ah, that's bad....